spinalcordtoolbox/scripts/sct_deepseg_sc.py

```diff
@@ -94,6 +94,10 @@ def get_parser():
         help='Output folder. Example: My_Output_Folder/ ',
         action=ActionCreateFolder,
         default=os.getcwd())
+    optional.add_argument(
+        "-o",
+        metavar=Metavar.file,
+        help='Output filename. Example: spinal_seg.nii.gz '),
     optional.add_argument(
         "-r",
```

Comment:
oh, except not this one line? but the other scripts all do use single quotes.

```suggestion
        '-r',
```